projects/mysadna2/src/app/app.component.scss

```diff
@@ -0,0 +1,16 @@
+.app-navigation {
+    float: right;
+    >a {
+        margin: 5px;
+        width: 83px;
+        height: 24px;
```

Comment:
Again, you took the width and height from zeplin, but it's not the best idea in a real page.
Zeplin mock is just a static model, kind of a picture. But website is a more dynamic.
Try to open your site in Inspect mode (Ctrl+Shift+C)
The rules just can't be handled:
https://i.imgur.com/SDqZkT1.png

When you're creating a mockup, especially responsive design, try to make blocks flexible. When its size and style can be changed from inside and outside.